### PR TITLE
manifest: bluetooth: mesh: health client fixes

### PR DESCRIPTION
**Pull request by m-alperen-sener**
Bluetooth: mesh: Health client fixes

Signed-off-by: Alperen Sener <alperen.sener@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 3c9414c179f4dcf8500d2c86f1ce6d6a7681afce
+      revision: pull/891/head
```
Comment (anangl) [inline west.yml:53]: Please update to `624271bddd622a4a0990d60b081da1e137950061`.

Comment (m-alperen-sener) [reply west.yml:53]: done

Comment (anangl) [reply west.yml:53]: Something went wrong. The branch was pushed but without the actual change.

Comment (m-alperen-sener) [reply west.yml]: Sorry, mistake. Got it right just now.

**Review by anangl — APPROVED** [review]

**Comment by m-alperen-sener** [comment]
It is covered with latest manifest update.

**Pull request closed without merging**